config/locales/de.yml

```diff
@@ -51,3 +51,8 @@ de:
         country: "Land"
         graduates_school_in: "Abitur vorraussichtlich in"
         already_graduated: "Bereits Abitur"
+  activerecord:
```

Comment:
Good idea, the newly created localizations weren't yet merged into dev. I moved them just now